WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -172,8 +173,8 @@ public static async Task ShellExecAsync(string cmd, bool waitForExit = true)
 			using var process = Process.Start(
 				new ProcessStartInfo
 				{
-					FileName = "/usr/bin/env",
-					Arguments = $"sh -c \"{escapedArgs}\"",
+					FileName = "/usr/bin/env sh",
+					Arguments = $"-c \"{escapedArgs}\"",
```

Comment:
This should change, similar to #4225

Reply:
At this point, we need to tackle https://github.com/zkSNACKs/WalletWasabi/pull/4215 first. I'll update this PR later. The conflict will be easy to spot.